lametro/bills.py

```diff
@@ -155,6 +158,24 @@ def scrape(self, window=28) :
             for topic in self.topics(matter_id) :
                 bill.add_subject(topic['MatterIndexName'].strip())
 
+            for relation in self.relations(matter_id):
+                try:
+                    # Get data (i.e., json) for the related bill. 
+                    # Then, we can find the 'MatterFile' (i.e., identifier) and the 'MatterIntroDate' (i.e., to determine its legislative session).
+                    # Sometimes, the related bill does not yet exist: in this case, throw an error, and continue.
+                    related_bill = self.endpoint('/matters/{0}', relation['MatterRelationMatterId'])
```

Comment:
The checking of existence seems like something you might want to push upstream to python-legistar.

Reply:
Okay....so, we could add a try-except block in the `endpoint` function: https://github.com/opencivicdata/python-legistar-scraper/blob/master/legistar/bills.py#L233

But I wonder if this is a "better-when-fragile" scenario? (At least, with the Metro bill relations, we know that the endpoint function often throw an error...but in other cases, maybe we want to see that error?)

Reply:
That's a fair argument. If we find that we always want to skip the cases where an error is thrown, then we can move it to python-legistar.